vuepress/docs/v6.3.2/docs/getting-started/README.md

```diff
@@ -8,9 +8,9 @@ sidebarDepth: 2
 **Mac:** Entando 6 is not currently compatible with the Apple M1 ARM64 architecture found in some newer Macs.
```

Comment:
I guess I thought we were just going to do this for /next but given that the 7 release is at least weeks away, I'm fine to add this to next as well once it's done.

Reply:
i wasn't sure if this warning would apply to both 6.3.2 and Next; i know our rule is to focus on Next outside of exceptions, but since 6.3.2 required an edit to address the M1 chip i thought i might as well also absorb the fixes meant for Next and use 6.3.2 as the version to modify/approve

Reply:
Works for me!